datachannel_js.go

```diff
@@ -243,6 +273,8 @@ func valueToDataChannelMessage(val js.Value) DataChannelMessage {
 			return js.Undefined()
 		}))
 
+		reader.Call("readAsArrayBuffer", val)
```

Comment:
I'm guessing the purpose of this call is to flush the buffer? Maybe this is worth a small comment becuase it's not immediately obvious.

Reply:
I think this was just missing from the conversion code. It's mentioned in the MDN docs linked a couple lines above:
https://developer.mozilla.org/en-US/docs/Web/API/Blob#Example_for_extracting_data_from_a_Blob

Without this line `val` is never passed to the reader. Maybe we never tested the Blob type in the end.